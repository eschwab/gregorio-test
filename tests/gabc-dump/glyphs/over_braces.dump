=====================================================================
 SCORE INFOS
=====================================================================
   number_of_voices          1


=====================================================================
 VOICES INFOS
=====================================================================
  Voice 1
   initial_key               5 (c3)


=====================================================================
 SCORE
=====================================================================
   type                      9 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    14 (GRE_BAR)
     bar                     2 (B_DIVISIO_MINIMA)
     special sign            22 (_H_EPISEMUS)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
=====================================================================
   type                      9 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
         h_episemus_type        3 (H_MULTI)
         h_episemus_top_note    g
---------------------------------------------------------------------
     type                    14 (GRE_BAR)
     bar                     2 (B_DIVISIO_MINIMA)
     special sign            22 (_H_EPISEMUS)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    h
=====================================================================
   type                      9 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    14 (GRE_BAR)
     bar                     2 (B_DIVISIO_MINIMA)
     special sign            17 (_V_EPISEMUS_H_EPISEMUS)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
=====================================================================
   type                      9 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
         h_episemus_type        3 (H_MULTI)
         h_episemus_top_note    g
---------------------------------------------------------------------
     type                    14 (GRE_BAR)
     bar                     2 (B_DIVISIO_MINIMA)
     special sign            17 (_V_EPISEMUS_H_EPISEMUS)
---------------------------------------------------------------------
     type                    13 (GRE_SPACE)
     space                   6 (SP_GLYPH_SPACE)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    h
=====================================================================
   type                      9 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
         TeX string             "\greoverbrace{6mm}{0pt}{0pt}{1}"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  13 (GRE_SPACE)
       space                 3 (SP_ZERO_WIDTH)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            23 (G_PORRECTUS)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
=====================================================================
   type                      9 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
         TeX string             "\greovercurlybrace{6mm}{0}{0pt}{0pt}{1}"
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  13 (GRE_SPACE)
       space                 3 (SP_ZERO_WIDTH)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            23 (G_PORRECTUS)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    12 (GRE_END_OF_LINE)
     sub_type                15 (GRE_END_OF_PAR)
=====================================================================
   type                      9 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
         TeX string             "\greovercurlybrace{6mm}{1}{0pt}{0pt}{1}"
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            23 (G_PORRECTUS)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
=====================================================================
