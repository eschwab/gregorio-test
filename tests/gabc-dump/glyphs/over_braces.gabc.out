=====================================================================
 SCORE INFOS
=====================================================================
   number_of_voices          1


=====================================================================
 VOICES INFOS
=====================================================================
  Voice 1
   initial_key               5 (c3)


=====================================================================
 SCORE
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    10 (GRE_BAR)
     element_type            2 (B_DIVISIO_MINIMA)
     additional_infos        2 (_H_EPISEMUS)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
         h_episemus_type        3 (H_MULTI)
         h_episemus_top_note    g
---------------------------------------------------------------------
     type                    10 (GRE_BAR)
     element_type            2 (B_DIVISIO_MINIMA)
     additional_infos        2 (_H_EPISEMUS)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    h
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    10 (GRE_BAR)
     element_type            2 (B_DIVISIO_MINIMA)
     additional_infos        2 (_V_EPISEMUS_H_EPISEMUS)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
         h_episemus_type        3 (H_MULTI)
         h_episemus_top_note    g
---------------------------------------------------------------------
     type                    10 (GRE_BAR)
     element_type            2 (B_DIVISIO_MINIMA)
     additional_infos        2 (_V_EPISEMUS_H_EPISEMUS)
---------------------------------------------------------------------
     type                    9 (GRE_SPACE)
     element_type            54 (SP_GLYPH_SPACE)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
         h_episemus_type        2 (H_ALONE)
         h_episemus_top_note    h
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         TeX string             "\greoverbrace{6mm}{0pt}{0pt}{1}"
         shape                  1 (S_PUNCTUM)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  9 (GRE_SPACE)
       glyph_type            51 (SP_ZERO_WIDTH)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            23 (G_PORRECTUS)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         TeX string             "\greovercurlybrace{6mm}{0}{0pt}{0pt}{1}"
         shape                  1 (S_PUNCTUM)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  9 (GRE_SPACE)
       glyph_type            51 (SP_ZERO_WIDTH)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            23 (G_PORRECTUS)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    8 (GRE_END_OF_LINE)
     element_type            13 (GRE_END_OF_PAR)
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style          ST_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style                ST_CENTER
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         TeX string             "\greovercurlybrace{6mm}{1}{0pt}{0pt}{1}"
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            23 (G_PORRECTUS)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  h
         shape                  1 (S_PUNCTUM)
=====================================================================
